framework/components/ema-content-slider/ema-content-slider.twig

```diff
@@ -0,0 +1,35 @@
+{#
+  Accepts:
+    -
+#}
+
+{# Internal properties #}
+{% set _css_class = 'ema-content-slider' %}
+
+
+{# Internal logic - Process properties #}
+{% if extra_class is defined %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_class %}
+{% endif %}
+
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~'"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print result #}
+<ul class="ecl-listing {{ _css_class }}" {{ _extra_attributes }}>
```

Comment:
@emeryro This is the main container of the content-slier. It has `_extra_attributes`

Reply:
ok, i guess I checked a previous version